CMakeLists.txt

```diff
@@ -756,16 +756,15 @@ if(IREE_BUILD_BENCHMARKS)
   add_subdirectory(benchmarks)
 endif()
 
+# tools/ can depend on compiler/ and runtime/
+add_subdirectory(tools)
```

Comment:
Maybe a note here that the ordering affects which phony targets are user visible? (mainly so that a future refactor doesn't move it without thinking)